modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -491,7 +583,7 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
 
         $where =" WHERE s.Active='Y'
             AND s.Current_stage <> 'Recycling Bin'
-            AND f.CommentID NOT LIKE '%DDE%'
+            AND f.CommentID NOT LIKE '%DDE%'  
```

Comment:
GitHub says that there's some unnecessary whitespace added to the end of this line.